src/version.h

```diff
@@ -38,8 +38,9 @@ static const int INVALID_CB_NO_BAN_VERSION = 70015;
 
 //! first odo version
 static const int ODO_FORK_VERSION = 70017;
+
 //! "wtxidrelay" command for wtxid-based relay starts with this version
-static const int WTXID_RELAY_VERSION = 70017;
+static const int WTXID_RELAY_VERSION = 70019;
```

Comment:
Correct me if I am wrong, but I think our next version is `70018`?

Reply:
Good catch. Yeah 70018  should be next version bump.

https://github.com/DigiByte-Core/digibyte/blob/develop/src/version.h#L13

Reply:
This was done as a preventative measure.
I didn't want to enable wtxid type relay whilst the legacy network is still up.
PROTOCOL_VERSION is currently 70018 (for 8.22), so i simply set it to 70019 so it doesn't become enabled.

Reply:
> This was done as a preventative measure.
> 
> I didn't want to enable wtxid type relay whilst the legacy network is still up.
> 
> PROTOCOL_VERSION is currently 70018 (for 8.22), so i simply set it to 70019 so it doesn't become enabled.

Sounds good!